cmd/virt-launcher/virt-launcher.go

```diff
@@ -300,6 +321,12 @@ func main() {
 	// managing virtual machines.
 	markReady(*readinessFile)
 
+	domainUUID := waitForDomainUUID(*qemuTimeout, domainManager)
```

Comment:
Why do you move it after `markReady`?

Reply:
The `NewProcessMonitor` call from above just created the monitor, it also started after the `markReady` call. The reason is, that right now we are waiting for that marker on the cluster before we even define the vm in virt-launcher.

Reply:
missed the fact that it only initialization, thanks